packages/rum-react/test/specs/get-with-transaction.spec.js

```diff
@@ -51,26 +52,30 @@ function TestComponent(apm) {
 }
 
 describe('withTransaction', function() {
+  const { serverUrl, serviceName } = getGlobalConfig().agentConfig
+  let apmBase, serviceFactory
+
+  beforeEach(() => {
+    serviceFactory = createServiceFactory()
+    apmBase = new ApmBase(serviceFactory, false)
+    apmBase.init({
+      active: true,
+      serverUrl,
+      serviceName,
+      disableInstrumentations: ['page-load', 'error']
+    })
+  })
+
   it('should work if apm is disabled or not initialized', function() {
-    TestComponent(new ApmBase(createServiceFactory(), true))
-    TestComponent(new ApmBase(createServiceFactory(), false))
+    TestComponent(new ApmBase(serviceFactory, true))
```

Comment:
This might be not related to this PR, but I'm not sure why this test passes. We probably should investigate why when we pass disable = true to ApmBase, it still calls startTransaction.

Do you happen to know?
Either way we can address that separately if you prefer.

Reply:
I guess it does not matter, component should rendered if APM is enabled/disabled and we don't check if startTransaction is called or not. So I am not sure what is the issue here?

Reply:
We probably can improve this test a bit, let's do it separately!